src/pages/studyView/table/ClinicalTable.tsx

```diff
@@ -35,89 +89,35 @@ export default class ClinicalTable extends React.Component<IClinicalTableProps,
     }
 
     @bind
-    @action private tooltipLabelMouseEnter(value: string): void {
+    @action
+    private tooltipLabelMouseEnter(value: string): void {
         if (this.props.highlightedRow) {
             this.props.highlightedRow(value);
         }
     }
 
     @bind
-    @action private tooltipLabelMouseLeave(): void {
+    @action
```

Comment:
action should probably be in the highlightedRow callback itself or whatever actually directly mutates the mobx observable